server/data/data-jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -26,6 +26,12 @@
     <persistence-unit name="James" transaction-type="JTA">
        <provider>org.apache.openjpa.persistence.PersistenceProviderImpl</provider>
         <jta-data-source>osgi:service/javax.sql.DataSource/(osgi.jndi.service.name=jdbc/james)</jta-data-source>
+        <class>org.apache.james.mailbox.jpa.mail.model.JPAMailbox</class>
```

Comment:
Of course, the persistence in Guice is enriched but the persistence has to contain all entities model so I think it till needs here.